code/datums/components/crafting/recipes.dm

```diff
@@ -1767,30 +1767,14 @@
 	name = "Toiletbong"
 	category = CAT_ENTERTAINMENT
 	tool_behaviors = list(TOOL_WRENCH)
-	reqs = list(/obj/item/flamethrower = 1)
-	machinery = list( /obj/structure/toilet)
+	reqs = list(/obj/item/stack/sheet/iron = 1,
+			    /obj/item/flamethrower = 1,
+				/obj/item/tank/internals/plasma = 1)
 	result = /obj/structure/toiletbong
 	time = 5 SECONDS
-	additional_req_text = " plasma tank (on flamethrower), toilet"
-
-/datum/crafting_recipe/toiletbong/check_requirements(mob/user, list/collected_requirements)
-	if((locate(/obj/structure/toilet) in range(1, user.loc)) == null)
-		return FALSE
-	var/obj/item/flamethrower/flamethrower = collected_requirements[/obj/item/flamethrower][1]
-	if(flamethrower.ptank == null)
```

Comment:
the recipe no longer requires the tank has plasma, is that intentional?

Reply:
Yes, I thought it would lower the risk of handing out flamethrowers to people;
I am fine if the plasma should stay in, either way is ok for me

Reply:
@JohnFulpWillard I would leave this to what your preference, I have no strong objections here

Reply:
I'm really sorry. @LordVollkorn ...I just made a PR for this without knowing you were already working on it. https://github.com/tgstation/tgstation/pull/72893

I went for the 'change crafting so it can support structures' route.

Reply:
@vinylspiders Don't worry about it, you are doing god's work there. All I wanted was for that crafting recipe to work again, does not matter who makes it happen. Also, I wouldn't want to let your structures code go to waste. I would leave it up to the maintainers to decide, and if they go for your PR I am 100% ok with that.
In any case, the structure code you made is valuable and should be PR'd anyway 🤟